calico_containers/tests/unit/calicoctl_test.py

```diff
@@ -216,6 +219,123 @@ def test_validate_arguments(self, case, sys_exit_called):
             self.assertEqual(m_sys_exit.called, sys_exit_called)
 
 
+    @patch('calico_ctl.node.os', autospec=True)
+    @patch('calico_ctl.node.check_system', autospec=True)
+    @patch('calico_ctl.node.get_host_ips', autospec=True)
+    @patch('calico_ctl.node.warn_if_unknown_ip', autospec=True)
+    @patch('calico_ctl.node.warn_if_hostname_conflict', autospec=True)
+    @patch('calico_ctl.node.install_kubernetes', autospec=True)
+    @patch('calico_ctl.node.client', autospec=True)
+    @patch('calico_ctl.node.docker_client', autospec=True)
+    @patch('calico_ctl.node.docker', autospec=True)
+    @patch('calico_ctl.node._find_or_pull_node_image', autospec=True)
+    def test_node_start(self, m_find_or_pull_node_image, m_docker,
+                        m_docker_client, m_client, m_install_kube,
+                        m_warn_if_hostname_conflict, m_warn_if_unknown_ip,
+                        m_get_host_ips, m_check_system, m_os):
+        """
+        Test node_start function in calicoctl node command
+        """
+        # Set up mock objects
+        m_os.path.exists.return_value = False
+        m_get_host_ips.return_value = ['1.1.1.1', '2.2.2.2', '3.3.3.3']
+        m_os.getenv.return_value = '1.1.1.1:80'
+        m_docker.utils.create_host_config.return_value = 'host_config'
+        m_docker_client.create_container.return_value = {'Id':666}
+
+        # Set up arguments
+        node_image = 'node_image'
+        log_dir = './log_dir'
+        ip = ''
+        ip6 = 'aa:bb::zz'
+        as_num = ''
+        detach = True
+        kubernetes = True
+
+        # Call method under test
+        node.node_start(
+            node_image, log_dir, ip, ip6, as_num, detach, kubernetes
+        )
+
+        # Set up variables used in assertion statements
+        environment = [
+            "HOSTNAME=%s" % node.hostname,
+            "IP=3.3.3.3",
+            "IP6=%s" % ip6,
+            "ETCD_AUTHORITY=1.1.1.1:80",  # etcd host:port
+            "FELIX_ETCDADDR=1.1.1.1:80",  # etcd host:port
+        ]
+        binds = {
+            "/proc":
+                {
+                    "bind": "/proc_host",
+                    "ro": False
+                },
+            log_dir:
+                {
+                    "bind": "/var/log/calico",
+                    "ro": False
+                },
+            "/run/docker/plugins":
+                {
+                    "bind": "/usr/share/docker/plugins",
+                    "ro": False
+                }
+        }
+
+        # Assert
+        m_os.path.exists.assert_called_once_with('./log_dir')
```

Comment:
Use your variable names for your asserts.  It is easier to maintain if the value of the variable changes.  Looks like you've already done this for most of your other asserts.